docs/source/examples/learning-depolarizing-noise.myst

```diff
@@ -124,14 +124,14 @@ import warnings
 # this example only simulates noise on CNOT
 warnings.simplefilter("ignore", UserWarning)
 loss = [
-    biased_noise_loss_function(
-        [eps, 0],
-        operations_to_learn,
-        training_circuits,
-        ideal_values,
-        noisy_executor,
-        pec_kwargs,
-        observable,
+    depolarizing_noise_loss_function(
+        epsilon=[eps],
+        operations_to_mitigate=operations_to_learn,
+        training_circuits=training_circuits,
+        ideal_values=ideal_values,
+        noisy_executor=noisy_executor,
+        pec_kwargs=pec_kwargs,
+        observable=observable,
     )
```

Comment:
The only one that is needed is `observable` because I skip specifying `pec_data`. I think any mention of `pec_data` in this example would be confusing.

Would the following be preferable in this case?

```
loss = [
    depolarizing_noise_loss_function(
       [eps],
       operations_to_learn,
       training_circuits,
       ideal_values,
       noisy_executor,
       pec_kwargs,
       observable=observable,
   )
   for eps in epsilons
]

```

Reply:
And should I do something similar when calling `learn_depolarizing_noise_parameters`?

Reply:
Sounds good to me!